### PR TITLE
Fix/code style

### PR DESCRIPTION
**Pull request by DaQuirm**
Numerous minor code style/syntax fixes.
Mind the following:
* ESLint check is now a part of the pre-commit script
* There are two warnings left (concerning dangling underscores), they won't prevent commits from succeeding but probably should be fixed (because, you know, getting warnings sucks :pouting_cat: )
* Specs are excluded temporarily because we might need a slightly different ruleset for them
* .editorconfig should now be in working order, try using it in your editors/IDEs :cat:

@mr-mig @dev-tim @larsblumberg

### CODE REVIEW

File: public/app/app.jsx [hunk 1]
```diff
@@ -1,32 +1,27 @@
...
 var MainScreen = require('./screens/main');
 var DevicesRegion = require('./screens/main/sections/devices');
 var SettingsRegion = require('./screens/main/sections/settings');
 
 var routes = (
-  <Route path="/" handler={MainScreen}>
-    <Route name="settings" handler={SettingsRegion}/>
-    <Route name="devices" handler={DevicesRegion}/>
-    <Redirect from="/" to="devices"/>
+  <Route path='/' handler={MainScreen}>
```
Comment (mr-mig) [inline public/app/app.jsx]: This one will make me (Oo), but it's ok

File: public/app/stores/navigation/navigation-store.js [hunk 2]
```diff
@@ -1,13 +1,12 @@
-var React = require('react');
 var Reflux = require('reflux');
 
 var _items = [
-  {title: "My devices", linkTo: "devices"},
-  {title: "My settings", linkTo: "settings"}
+  { title: 'My devices', linkTo: 'devices' },
```
Comment (mr-mig) [inline public/app/stores/navigation/navigation-store.js]: That's nice

**Comment by mr-mig** [comment]
LGTM

File: public/app/app.jsx (hunk 1 above)
Comment (DaQuirm) [inline public/app/app.jsx]: You mean single quotes in the markup?

File: public/app/stores/navigation/navigation-store.js (hunk 2 above)
Comment (DaQuirm) [inline public/app/stores/navigation/navigation-store.js]: I know, so lovely :heart:

File: public/app/app.jsx (hunk 1 above)
Comment (mr-mig) [inline public/app/app.jsx]: yep

**Comment by dev-tim** [comment]
LGTM!

File: public/app/actions/actions.js [hunk 3]
```diff
@@ -4,10 +4,10 @@ var actions = Reflux.createActions([
   // user actions
   'addDevice',
   'removeDevice',
-  //room actions
+  // room actions
   'addRoom',
   'removeRoom',
-  //settings actions
+  // settings actions
```
Comment (larsblumberg) [inline public/app/actions/actions.js]: +1 for adding a space between `//` and the comment text. Would love if everybody does it as it's more readible

File: public/app/app.jsx (hunk 1 above)
Comment (larsblumberg) [inline public/app/app.jsx]: Why we aren't using double quotes in the markup anymore? This somewhat helped distinguishing between js code and jsx code

Comment (DaQuirm) [inline public/app/app.jsx]: I'd say syntax highlighting is quite sufficient for distinguishing purposes. Personally I have nothing against double quotes in markup, but there seems to be no way to configure ESLint to distinguish between js and jsx.

**Pull request merged**